rust/feature-flags/src/database_pools.rs

```diff
@@ -33,7 +33,12 @@ impl DatabasePools {
                 None
             },
             max_lifetime: if config.max_lifetime_secs > 0 {
-                Some(Duration::from_secs(config.max_lifetime_secs))
+                let jitter = if config.max_lifetime_jitter_secs > 0 {
+                    rand::random::<u64>() % config.max_lifetime_jitter_secs
```

Comment:
This works fine, but there's a method specifically for this for idiomatic uniform distribution. It makes the intent a little more clear and `gen_range` doesn't have the Modulo bias (completely negligible for this scenario, but good habits and all that).

```suggestion
                    rand::thread_rng().gen_range(0..config.max_lifetime_jitter_secs)
```

You'll need to import: `use rand::Rng;` though.